lib/engine.js

```diff
@@ -38,13 +38,20 @@ Engine = function(initializer) {
   engine = this;
   isStarted = false;
   engine.entityList = Lill.attach({});
-  engine.addEntity = function(components) {
-    var entity;
-    entity = Entity(components);
+  engine.addEntity = function(entity) {
+    if (!entity) {
+      entity = new Entity;
+    } else if (entity instanceof Array) {
+      log('adding Entity as an array of components should be done using `buildEntity` instead');
```

Comment:
Please change it to: `Passing array of components to addEntity method is deprecated. Use buildEntity method instead.`